presto-cli/src/main/java/io/prestosql/cli/TableNameCompleter.java

```diff
@@ -58,10 +58,18 @@ public TableNameCompleter(QueryRunner queryRunner)
 
     private List<String> listTables(String schemaName)
     {
-        return queryMetadata(format("SELECT table_name FROM information_schema.tables WHERE table_schema = '%s'", schemaName));
+        String[] array = schemaName.split("\\.");
+        if (array.length == 1) {
+            return queryMetadata(format("SELECT table_schema || '.' || table_name FROM %s.information_schema.tables", array[0]));
```

Comment:
enclose schema name in `"`-quotes:

```suggestion
            return queryMetadata(format("SELECT table_schema || '.' || table_name FROM \"%s\".information_schema.tables", array[0].replace("\"", "\"\"")));
```

Reply:
Fixed to enclose.